src/WebJobs.Script/Config/HostJsonFileConfigurationSource.cs

```diff
@@ -198,7 +198,7 @@ internal JObject LoadHostConfig(string configFilePath)
                     JObject hostConfigObject;
                     try
                     {
-                        string json = File.ReadAllText(configFilePath);
+                        string json = FileUtility.ReadAllText(configFilePath);
```

Comment:
Why the change here?

Reply:
This is the requirement of the Integration testing.  
The current integration testing (not my update) , it mock the reading file operation when it use it in integration testing. `FileUtility` is simply delegate to File in production, for testing, it modify the request from test fixture.